ingest/build-configs/ncbi/rules/curate.smk

```diff
@@ -0,0 +1,168 @@
+"""
+This part of the workflow handles the curation of data from NCBI
+
+REQUIRED INPUTS:
+
+    ndjson      = ncbi/data/ncbi.ndjson
+
+OUTPUTS:
+
+    metadata    = ncbi/results/metadata.tsv
+    seuqences   = ncbi/results/sequences_{segment}.fasta
+
+"""
+
+
+rule fetch_general_geolocation_rules:
+    output:
+        general_geolocation_rules="ncbi/data/general-geolocation-rules.tsv",
+    params:
+        geolocation_rules_url=config["curate"]["geolocation_rules_url"],
+    shell:
+        """
+        curl {params.geolocation_rules_url} > {output.general_geolocation_rules}
+        """
+
+
+rule concat_geolocation_rules:
+    input:
+        general_geolocation_rules="ncbi/data/general-geolocation-rules.tsv",
+        local_geolocation_rules=config["curate"]["local_geolocation_rules"],
+    output:
+        all_geolocation_rules="ncbi/data/all-geolocation-rules.tsv",
+    shell:
+        """
+        cat {input.general_geolocation_rules} {input.local_geolocation_rules} >> {output.all_geolocation_rules}
```

Comment:
why the `>>` instead of `>` ?

Reply:
Good question. This was copied from [pathogen-repo-guide](https://github.com/nextstrain/pathogen-repo-guide/blob/80cc1733b47f173772fe45005b82a69c6f8cb6c9/ingest/rules/curate.smk#L39), which was based on [mpox/ingest](https://github.com/nextstrain/mpox/blob/7f5adc3cab0c3034e455882581cc081f26f5ebbe/ingest/rules/curate.smk#L35) and leads back to [my own commit from ~2 years ago](https://github.com/nextstrain/mpox/commit/67aa5f2cbb08b9ef0d3ec75b78518786d59f7cbe). 

What was I thinking 🙃 My best guess is this is left over from when I had a rule to just append the `local_geolocation_rules` to the existing `general_geolocation_rules` without creating the additional `all_geolocation_rules` file.

Reply:
Updated in https://github.com/nextstrain/avian-flu/pull/40/commits/096ce904b8599227376294d4a691ad4ecde8ef61